File: aufgaben/02_tools_automation/04_abhaengigkeiten/typescript/main.ts
import axios from "axios";
import * as dayjs from "dayjs";

async function run() {
  try {
    const res = await axios.get("https://icanhazdadjoke.com/", {
      headers: { Accept: "text/plain" },
    });

    console.log("Dad Joke:", res.data);
    console.log("Timestamp:", dayjs().format("YYYY-MM-DD HH:mm:ss"));
  } catch (err) {
    console.error("es kracht:", err);
  }
}

run();

File: aufgaben/02_tools_automation/04_abhaengigkeiten/python/main.py
import requests
import numpy as np


url = "https://icanhazdadjoke.com/"
headers = {'Accept': 'text/plain'}

response = requests.get(url, headers=headers)

if response.status_code == 200:
    print("Dad Joke:", response.text)
else:
    print("oh oh... failed to fetch dad joke!")


a = np.array([1, 2, 3])
print(a * 2)
